apps/meteor/client/views/admin/apps/AllAppsSection.tsx

```diff
@@ -0,0 +1,24 @@
+import { App } from '@rocket.chat/core-typings';
```

Comment:
I think I removed this file last PR are we willing to reintroduce it again?
we are not even using it in the pull request

Reply:
Solved